app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/CheckboxInput/__test__/CheckBoxInput.test.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+import CheckBoxInput from 'components/WranglerGrid/SelectColumnPanel/DataTable/CheckboxInput';
+
+describe('It should render ', () => {
+  const mockMultipleSelect = jest.fn();
+  it('Should render the checkBox component and click the checkbox with disable check false', () => {
```

Comment:
This just checks if a checkbox can be checked??

Reply:
resolved.

Reply:
we have removed this test file
this did not need to be a separate component
moved the code to its parent 
following is the updated test case
1. now on clicking the checkbox, we are checking whether the element state is updated to checked or not
2. but the main test is to check whether the checkbox is rendered instead of radio based on the conditions passed from the parent

code for reference - https://github.com/cdapio/cdap-ui/pull/789/files#:~:text=it(%27Should%20render%20Radio%20Componene%20when%20%22isSingleSelection%22%20is%20true%27%2C%20()%20%3D%3E%20%7B

hope this is fine
plz, confirm @seanfeld